### PR TITLE
Update according CI specs

### PR DESCRIPTION
**Pull request by tonyredondo**
Update environment variables parser to follow the latest specs.

### CODE REVIEW

File: internal/utils/ci_providers.go [hunk 1]
```diff
@@ -58,17 +58,39 @@ func GetProviderTags() map[string]string {
...
+	tags[constants.GitCommitSHA] = getEnvironmentVariableIfIsNotEmpty("DD_GIT_COMMIT_SHA", tags[constants.GitCommitSHA])
+	tags[constants.GitCommitMessage] = getEnvironmentVariableIfIsNotEmpty("DD_GIT_COMMIT_MESSAGE", tags[constants.GitCommitMessage])
+	tags[constants.GitCommitAuthorName] = getEnvironmentVariableIfIsNotEmpty("DD_GIT_COMMIT_AUTHOR_NAME", tags[constants.GitCommitAuthorName])
+	tags[constants.GitCommitAuthorEmail] = getEnvironmentVariableIfIsNotEmpty("DD_GIT_COMMIT_AUTHOR_EMAIL", tags[constants.GitCommitAuthorEmail])
+	tags[constants.GitCommitAuthorDate] = getEnvironmentVariableIfIsNotEmpty("DD_GIT_COMMIT_AUTHOR_DATE", tags[constants.GitCommitAuthorDate])
+	tags[constants.GitCommitCommitterName] = getEnvironmentVariableIfIsNotEmpty("DD_GIT_COMMIT_COMMITTER_NAME", tags[constants.GitCommitCommitterName])
+	tags[constants.GitCommitCommitterEmail] = getEnvironmentVariableIfIsNotEmpty("DD_GIT_COMMIT_COMMITTER_EMAIL", tags[constants.GitCommitCommitterEmail])
+	tags[constants.GitCommitCommitterDate] = getEnvironmentVariableIfIsNotEmpty("DD_GIT_COMMIT_COMMITTER_DATE", tags[constants.GitCommitCommitterDate])
```
Comment (AdrianLC) [inline internal/utils/ci_providers.go:74]: this could be extracted to a function

File: internal/utils/ci_providers.go [hunk 2]
```diff
@@ -110,10 +132,14 @@ func extractAppveyor() map[string]string {
 	tags[constants.CIProviderName] = "appveyor"
 	if os.Getenv("APPVEYOR_REPO_PROVIDER") == "github" {
 		tags[constants.GitRepositoryURL] = fmt.Sprintf("https://github.com/%s.git", os.Getenv("APPVEYOR_REPO_NAME"))
-		tags[constants.GitCommitSHA] = os.Getenv("APPVEYOR_REPO_COMMIT")
-		tags[constants.GitBranch] = firstEnv("APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH", "APPVEYOR_REPO_BRANCH")
-		tags[constants.GitTag] = os.Getenv("APPVEYOR_REPO_TAG_NAME")
+	} else {
+		tags[constants.GitRepositoryURL] = os.Getenv("APPVEYOR_REPO_NAME")
```
Comment (AdrianLC) [inline internal/utils/ci_providers.go:136]: does this make sense? when it's not github it's always a URL?

Comment (tonyredondo) [reply internal/utils/ci_providers.go:136]: good point, this is based on the current implementation on other languages, we are handling only for github.

According to: https://www.appveyor.com/docs/environment-variables/

Appveyor support the following providers:

```
APPVEYOR_REPO_PROVIDER - gitHub, bitBucket, kiln, vso, gitLab, gitHubEnterprise, gitLabEnterprise, stash, gitea, git, mercurial or subversion
```

Maybe we should also add the support bitBucket and gitLab. But we should do that as a global issue for all tracers, so I'll leave that to another PR.

File: internal/utils/ci_providers.go (hunk 1 above)
Comment (tonyredondo) [reply internal/utils/ci_providers.go]: done in 372884c

**Review by juan-fernandez — APPROVED** [review]

**Pull request merged**